### ISSUE TITLE
Missing focus indicator for 'Block I' link at mobile viewport width (accessibility)

### ISSUE THREAD

**Issue by dandalpiaz** [web-illinois/ilw-header#31]
When viewing at a small viewport width (small enough so that the 'UNIVERSITY OF ILLINOIS URBANA-CHAMPAIGN' link is hidden at the top of header), the 'Block I' image link does not have a visual focus indicator when the link has focus. There is no outline or color change when focused. 

Example screenshot below taken from https://builder3.toolkit.illinois.edu/index.html:

<img width="498" height="207" alt="Image" src="https://github.com/user-attachments/assets/73b7e9b6-5bbb-432a-8b53-b5c7676e8097" />

**Comment by sydbeth** [comment]
The designers added what this should look like in the Figma Theme library: https://www.figma.com/design/MgeLbeXT7ErBgUEn38QvfQ/Illinois-Theme-Library?node-id=9652-41510&m=dev

<img width="506" height="320" alt="Image" src="https://github.com/user-attachments/assets/5a6a2610-90f0-4447-8fa7-352769c6c964" />

Note that the dropshadow was a mistake, just the color and outline would change for the block I on mobile
